### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by MikKut**
Some test changes

### CODE REVIEW

**Comment by MikKut** [comment]
What's up?

File: CulturalShare.Gateway/Middleware/MiddlewareClasses/HandlingExceptionsMiddleware.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace CulturalShare.Gateway.Middleware.MiddlewareClasses;
 
 public class HandlingExceptionsMiddleware
 {
-    public RequestDelegate _next { get; }
+    private RequestDelegate _next { get; }
```
Comment (YakovAkk) [inline CulturalShare.Gateway/Middleware/MiddlewareClasses/HandlingExceptionsMiddleware.cs:9]: hi

Comment (MikKut) [reply CulturalShare.Gateway/Middleware/MiddlewareClasses/HandlingExceptionsMiddleware.cs:9]: hi

File: CulturalShare.Gateway/Middleware/MiddlewareClasses/HandlingExceptionsMiddleware.cs [hunk 2]
```diff
@@ -55,7 +55,7 @@ private static string GetError(Exception exception)
     }
     private static IReadOnlyDictionary<string, string[]> GetErrors(Exception exception)
     {
-        IReadOnlyDictionary<string, string[]> errors = null;
+        IReadOnlyDictionary<string, string[]> errors = null!;
```
Comment (YakovAkk) [inline CulturalShare.Gateway/Middleware/MiddlewareClasses/HandlingExceptionsMiddleware.cs:58]: hi 2

**Review by YakovAkk — APPROVED** [review]
ok you can merge

**Pull request merged**